README.md

```diff
@@ -1,11 +1,60 @@
 # webdriverio-appium-app-browserstack
-[WebdriverIO](http://webdriver.io/) Integration with BrowserStack.
+This repository demonstrates how to run Appium tests using [WebdriverIO](http://webdriver.io/) on BrowserStack App Automate.
 
-![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
-
-<img src = "http://webdriver.io/images/webdriverio.png" height = "100">
+<div align="center">
+<img src = "https://www.browserstack.com/images/layout/browserstack-logo-600x315.png" > <br>
+<img src = "https://webdriver.io/img/webdriverio.png"  height="140px">
+</div>
 
 Code samples to get started with Appium tests for your Native App using WebdriverIO.
 
-## Resources
-* [Get Started guide](https://www.browserstack.com/app-automate/appium-webdriverio)
+## Setup
+
+### Requirements
+* Node.js
+  - Install at least **v8.11.2 or higher** as this is the oldest active LTS version
+  - If you don't have Node installed, you can get get it installed from [here](https://nodejs.org/en/)
+
+### Install Dependencies
+To install the dependencies for Android tests, run :
+```sh
+cd android
+npm i
+```
+Or,
+To install the dependencies for iOS tests, run :
+```sh
+cd ios
+npm i
+```
+
+## Getting Started
+
+Getting Started with Appium tests using WebdriverIO on BrowserStack couldn't be easier!
+
+### Run first test:
+  - Switch to `run_first_test` directory under [Android examples](./android) or [iOS examples](./ios)
```

Comment:
change this, since now it has changed to run-first-test. There will be other places as well, where you have to make these changes